### PR TITLE
[docs] Fix several issues encountered during publishing 

### PR DESCRIPTION
**Pull request by ras0219-msft**

### CODE REVIEW

File: docs/users/manifests.md [hunk 1]
```diff
@@ -69,8 +68,9 @@ Manifests follow strict JSON: they can't contain C++-style comments (`//`) nor t
 you can use field names that start with `$` to write your comments in any object that has a well-defined set of keys.
 These comment fields are not allowed in any objects which permit user-defined keys (such as `"features"`).
 
-Each manifest contains a top level object with the fields documented below; the most important ones are
-[`"name"`](#name), the [version fields](#version-fields), and [`"dependencies"`](#dependencies):
+The latest JSON Schema is available at https://raw.githubusercontent.com/microsoft/vcpkg/master/scripts/vcpkg.schema.json. IDEs with JSON Schema support such as Visual Studio and Visual Studio Code can use this file to provide intellisense and syntax checking. For most IDEs, you should set `"$schema"` in your `vcpkg.json` to this URL (like the above example).
```
Comment (vicroms) [inline docs/users/manifests.md:71]:
```suggestion
The latest JSON Schema is available at https://raw.githubusercontent.com/microsoft/vcpkg/master/scripts/vcpkg.schema.json. IDEs with JSON Schema support such as Visual Studio and Visual Studio Code can use this file to provide IntelliSense and syntax checking. For most IDEs, you should set `"$schema"` in your `vcpkg.json` to this URL (like the above example).
```

**Review by vicroms — DISMISSED** [review]

File: docs/users/manifests.md [hunk 2]
```diff
@@ -69,8 +68,9 @@ Manifests follow strict JSON: they can't contain C++-style comments (`//`) nor t
 you can use field names that start with `$` to write your comments in any object that has a well-defined set of keys.
 These comment fields are not allowed in any objects which permit user-defined keys (such as `"features"`).
 
-Each manifest contains a top level object with the fields documented below; the most important ones are
-[`"name"`](#name), the [version fields](#version-fields), and [`"dependencies"`](#dependencies):
+The latest JSON Schema is available at https://raw.githubusercontent.com/microsoft/vcpkg/master/scripts/vcpkg.schema.json. IDEs with JSON Schema support such as Visual Studio and Visual Studio Code can use this file to provide IntelliSense and syntax checking. For most IDEs, you should set `"$schema"` in your `vcpkg.json` to this URL (like the above example).
```
Comment (Cheney-W) [inline docs/users/manifests.md:71]: https://raw.githubusercontent.com/microsoft/vcpkg/master/scripts/vcpkg.schema.json

Is the link written like this on purpose? Make the contents of the file appear on the web page?

Comment (Cheney-W) [reply docs/users/manifests.md:71]: Ping @ras0219-msft

Comment (ras0219-msft) [reply docs/users/manifests.md:71]: The link is written out on purpose because users are expected to copy it into their files.

**Review by vicroms — APPROVED** [review]

**Pull request merged**